docs/UserGuide.md

```diff
@@ -8,15 +8,19 @@
 
 {Give steps to get started quickly}
 
-1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+1. Ensure that you have _Java 11_ or above installed. The link to _Java 11_ installer is [here](https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html).
+2. Download the latest version of `Mod Happy` from [here](http://link.to/duke).
+3. Copy the jar file into an empty folder.
+4. Open a terminal on your laptop, and go to the working directory where the file is saved.
+5. Run the command  `java -jar tp.jar` to start the program.
+6. You can now enter different commands.
 
 ## Features
 
-Note:</br>
-Compulsory Flags start with "/". </br>
-Optional Flags start with "-". </br>
-Compulsory Parameters are fully capitalised: e.g. MODULE_CODE. </br>
+Note:<br>
+Compulsory Flags start with "/". <br>
+Optional Flags start with "-". <br>
+Compulsory Parameters are fully capitalised: e.g. MODULE_CODE. <br>
 Optional Parameters are in square brackets: e.g. [-m "MODULE_DESCRIPTION"]
```

Comment:
Minor nitpick but let's get rid of the unnecessary capitalisation in "Flags" and "Parameters" -> "flags" and "parameters". Let's also be consistent with whether the example ends with a . or not.